### PR TITLE
docs: add readme changelog and component state

### PR DESCRIPTION
**Pull request by matthprost**
## Type

- Enhancement

#### What is expected?

I added changelog and get started documentation based on README.md and CHANGELOG.md. I also added a new page "Components State" that will automatically get all stories of all components and shows which ones are deprecated.

#### The following changes where made:

Screenshot 2022-11-23 at 17 53 21
Screenshot 2022-11-23 at 17 55 16
Screenshot 2022-11-23 at 17 57 50

### CODE REVIEW

File: src/__stories__/ComponentState.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+Promise.allSettled(
+  componentsNames.map(
+    name => import(`../components/${name}/__stories__/index.stories`),
+  ),
+)
+  .then(module => {
+    modules = module
+  })
+  .catch(() => {})
```
Comment (chambo-e) [inline src/__stories__/ComponentState.tsx:24]: No other way to retrieve this with storybook ?

File: src/__stories__/Index.stories.mdx [hunk 2]
```diff
@@ -1,38 +1,6 @@
...
-      </Text>
-    </ThemeProvider>
-  )
-}
-```
-
-### Customizing
-
-The default theme is available on Github, you can see it [here](https://github.com/scaleway/scaleway-ui/blob/main/src/theme/index.ts). For more customization go to [Colors page](/docs/theme-colors--palette)
+<Description markdown={Readme} />
```
Comment (chambo-e) [inline src/__stories__/Index.stories.mdx:6]: No way to interpret emoji ? 

Also why remove the quickstart ? Changelog should be on another page no ?

File: src/__stories__/ComponentState.tsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    title: string
+    parameters: {
+      deprecated: boolean
+    }
+  }
+}>[]
+
+Promise.allSettled(
+  componentsNames.map(
+    name => import(`../components/${name}/__stories__/index.stories`),
```
Comment (QuiiBz) [inline src/__stories__/ComponentState.tsx:18]: Storybook doesn't supports top-level await?

**Review by QuiiBz — APPROVED** [review]

File: src/__stories__/ComponentState.tsx (hunk 3 above)
Comment (matthprost) [reply src/__stories__/ComponentState.tsx:18]: @QuiiBz it doesn't I couldn't do it without errors

File: src/__stories__/Changelog.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import { Meta, Description } from '@storybook/addon-docs'
+import Changelog from '../../CHANGELOG.md'
+
+<Meta title="Changelog" />
+
+# Changelog
+
+<Description markdown={Changelog} />
```
[the remark this answers is not in the document]
Comment (matthprost) [reply src/__stories__/Changelog.stories.mdx:8]: @chambo-e I've found out this package: https://github.com/joypixels/emoji-toolkit

It seems to be working just fine:
Screenshot 2022-11-30 at 11 58 30

File: src/__stories__/ComponentState.tsx (hunk 1 above)
Comment (matthprost) [reply src/__stories__/ComponentState.tsx:24]: I couldn't find any :(

**Review by ModuloM — APPROVED** [review]

**Pull request merged**